ur_driver/test_move.py

```diff
@@ -91,6 +93,10 @@ def main():
     global client
     try:
         rospy.init_node("test_move", anonymous=True, disable_signals=True)
+        prefix = rospy.get_param("/ur_driver/prefix", "")
```

Comment:
This should probably just read the parameter in the nodes private namespace, instead of hardcoding it like this.